gcs/CONFIGURATION.md

```diff
@@ -36,7 +36,14 @@
 *   `fs.gs.delegation.token.binding` (not set by default)
 
     Delegation Token binding class.
+*   `fs.gs.delegation.token.instantiation_strategy` (default `INSTANCE_PER_SERVICE`)
 
+    Defines GCS delegation token instantiation strategy.
+
+    Valid values:
+      * `INSTANCE_PER_SERVICE`  New token instance is created for each service instance;
+      * `SHARED` -  Token instance is shared across the instances of GCS services. 
+         Different renewers still get unique token instances;
```

Comment:
Add new lines before and after this config doc.

Reply:
done